src/lib/viewers/box3d/model3d/Model3DViewer.js

```diff
@@ -43,9 +43,21 @@ class Model3DViewer extends Box3DViewer {
     /** @property {Object[]} - List of Box3D instances added to the scene */
     instances = [];
 
-    /** @property {boolean} - Boolean indicating whether the animation is playihng */
+    /** @property {boolean} - Boolean indicating whether the animation is playing */
     isAnimationPlaying = false;
 
+    /** @property {boolean} - Boolean indicating whether the grid is showing */
+    renderGrid = DEFAULT_RENDER_GRID;
```

Comment:
The term `render` is usually used as a verb, which makes these look like methods at first. How about something like `currentGrid`?